packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -1073,48 +1204,23 @@ class FlameChartNode<T extends FlameChartDataMixin<T>> {
     // TODO(kenz): this is intended to improve performance but can probably be
     // improved. Perhaps we should still show a solid line and fade it out?
     if (zoomedWidth < 0.5) {
-      return SizedBox(width: zoomedWidth);
+      return;
```

Comment:
is this one of the bigger performance grabs in this PR?

Reply:
Hard to say its just one thing, really we're cutting out a ton of simple configuration and replacing it with the canvas calls it boils down to. So the real work we are doing should be about the same but there is less stuff in-between (multiple layout/paint calls et cetera) - if that makes sense